crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -548,6 +578,9 @@ impl SequenceAction {
 pub struct TransferAction {
     pub to: Address,
     pub amount: u128,
+    /// optional asset to be transferred.
+    /// if None, then the native asset is used.
```

Comment:
Noted above, I'd prefer if we didn't special case the native token but used it explicitly

Reply:
removed